packages/utilities/psammead-locales/moment/bn.js

```diff
@@ -2,7 +2,7 @@ var moment = require('moment');
 require('moment/locale/bn');
 
 moment.updateLocale('bn', {
-  months: 'ানুয়ারী_ফেব্রুয়ারি_মার্চ_এপ্রিল_মে_জুন_জুলাই_অগাস্ট_সেপ্টেম্বর_অক্টোবর_নভেম্বর_ডিসেম্বর'.split(
+  months: 'জানুয়ারি_ফেব্রুয়ারি_মার্চ_এপ্রিল_মে_জুন_জুলাই_অগাস্ট_সেপ্টেম্বর_অক্টোবর_নভেম্বর_ডিসেম্বর'.split(
```

Comment:
```suggestion
  months: 'জানুয়ারি_ফেব্রুয়ারি_মার্চ_এপ্রিল_মে_জুন_জুলাই_অগাস্ট_সেপ্টেম্বর_অক্টোবর_নভেম্বর_ডিসেম্বর'.split(
```

Reply:
The suggested change looks the same as what it was before or am I missing something?

Reply:
It's a bit odd, if I copy the word from your comment I can't find (Ctrl + F) it in my local version of the branch. If I copy the word from the code, I can Ctrl + F it in this PR. It could be issues with the language implementation in MacOS though.